### PR TITLE
Add Contexture

### PR DESCRIPTION
**Pull request by ltchris**
https://github.com/ltchris/awesome-contexture

This awesome list covers the best and the brightest of the Contexture ecosystem. Contexture is used for abstracting queries/filters and results/aggregrations from different backing data stores like ElasticSearch and MongoDB.

This list includes the best applications, libraries, documentation, and other resources from the Contexture ecosystem. It is a great way for the community to see the best of Contexture in one place and to be able to contribute to the growing ecosystem. 

Reviewed PRs:
https://github.com/sindresorhus/awesome/pull/1725#issuecomment-596160653
https://github.com/sindresorhus/awesome/pull/1722#issuecomment-596160909
https://github.com/sindresorhus/awesome/pull/1728#issuecomment-596244280
https://github.com/sindresorhus/awesome/pull/1721#issuecomment-596244766
https://github.com/sindresorhus/awesome/pull/1720#issuecomment-596245053
https://github.com/sindresorhus/awesome/pull/1703#issuecomment-596245422
https://github.com/sindresorhus/awesome/pull/1701#issuecomment-596245680
https://github.com/sindresorhus/awesome/pull/1699#issuecomment-596246004
https://github.com/sindresorhus/awesome/pull/1679#issuecomment-596247870
https://github.com/sindresorhus/awesome/pull/1676#issuecomment-596248005
https://github.com/sindresorhus/awesome/pull/1666#issuecomment-596248275
https://github.com/sindresorhus/awesome/pull/1660#issuecomment-596248493
https://github.com/sindresorhus/awesome/pull/1622#issuecomment-596248739

Passed `awesome-lint`.

By submitting this pull request I confirm I've read and complied with the [requirements](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) :vulcan_salute:

### CODE REVIEW

**Comment by ltchris** [comment]
unicorn

**Comment by simskij** [comment]
image

This is only my opinion of course, but the top section really looks awful as it is structured currently. Given the size of your image, I'd suggest you place it centered over the level 1 header.

I also think that you might gone a bit overboard with the description. Would it be possible to condense it and link to a more lengthy description?

**Comment by ltchris** [comment]
Thanks @simskij, I will try to condense the description and try the different image placement.

File: readme.md [hunk 1]
```diff
@@ -448,6 +448,7 @@
 - [CouchDB](https://github.com/quangv/awesome-couchdb#readme) - Document-oriented NoSQL database.
 - [HBase](https://github.com/rayokota/awesome-hbase#readme) - Distributed, scalable, big data store.
 - [NoSQL Guides](https://github.com/erictleung/awesome-nosql-guides#readme) - Help on using non-relational, distributed, open-source, and horizontally scalable databases.
+- [Contexture](https://github.com/ltchris/awesome-contexture) - Abstracting queries/filters and results/aggregrations from different backing data stores like ElasticSearch and MongoDB.
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
- [Contexture](https://github.com/ltchris/awesome-contexture#readme) - Abstracts queries/filters and results/aggregations from different backing data stores like ElasticSearch and MongoDB.
```

Comment (sindresorhus) [reply readme.md]: And the description in your readme should be:

> Contexture abstracts queries/filters and results/aggregations from different backing data stores like ElasticSearch and MongoDB.

**Comment by ltchris** [comment]
Thanks @sindresorhus, fixed!

**Comment by sindresorhus** [comment]
Not sure why you removed the linkified word [here](https://github.com/ltchris/awesome-contexture/commit/8b1e7a6834d873110ad6e1e0aa6b16f08c15e099), but that should stay.

**Comment by ltchris** [comment]
> Not sure why you removed the linkified word here, but that should stay.

Fixed.

**Comment by sindresorhus** [comment]
Also use proper commit titles in your repo. `Update readme.md` is quite useless (and noisy) when looking at the commit history.

**Comment by ltchris** [comment]
Thanks @sindresorhus, I agree that was bad practice.

Updated the [commit messages](https://github.com/ltchris/awesome-contexture/commits/master) to be more useful.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1237011247087808512

**Comment by sindresorhus** [comment]
I would just squash all the commits into one commit as they're not really useful.

**Pull request merged**